testsuite/probe/bcache1-devicegraph.xml

```diff
@@ -166,14 +166,14 @@
       <uuid>6990d6be-8fa8-4be7-8cf3-66ad1448864e</uuid>
       <mountpoint>swap</mountpoint>
       <mount-by>uuid</mount-by>
-      <fstab-options>defaults</fstab-options>
+      <mount-opts>defaults</mount-opts>
```

Comment:
Right. It is filtered out in the next step anyway, but there is no good reason to keep this here in the XML files.

Removed here and in 2 other XML files.